Makefile

```diff
@@ -1,8 +1,12 @@
 # Makefile
 
-run:
+server:
 	go get -d
-	go run *.go
+	go run main.go
+
+client:
+	go get -d
```

Comment:
If you combine server and client in same repository, then you should  have similar place for both main functions.
You can do as adviced here https://github.com/golang-standards/project-layout/tree/master/cmd